curriculum/locales/english/learn-digital-ledgers-by-building-a-blockchain.md

```diff
@@ -1,3 +1,4404 @@
 # Web3 - Learn Digital Ledgers by Building a Blockchain
 
 ## 1
+
+### --description--
+
+For this project, you will work in the `learn-digital-ledgers-by-building-a-blockchain` folder. Open a bash terminal and use `cd` to change into it.
+
+The tests should run automatically. If they don't, close any open **bash** terminals, open a new one, and run the command again.
```

Comment:
Yea, I was a little worried about that. I'm gonna leave it for now. This will be the first step on all these projects of mine - you may want to do it as well. I may reword all of them to something better later.